src/shared/services/commandInterpreterHelper.js

```diff
@@ -90,7 +91,7 @@ const availableCommands = [
     name: 'set-params',
     match: cmd => /^params?\s/.test(cmd),
     exec: function (action, cmdchar, put, store) {
-      handleParamsCommand(action, cmdchar, put)
+      return handleParamsCommand(action, cmdchar, put, bolt)
```

Comment:
It looks like `handleParamsCommand ` isn't using the '`bolt` argument so it can be skipped here (and also the import up top)